files/zabbix/selinux-zabbix.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Look at getting the `[root@lark-zabbix-01 ~]# setsebool -P httpd_can_network_connect on` configured through Puppet like the other SeLinux stuff you solved through Puppet.